tests/e2e/improved_csi_idempotency.go

```diff
@@ -344,6 +514,55 @@ func createVolumeWithServiceDown(serviceName string, namespace string, client cl
 
 		ginkgo.By(fmt.Sprintf("Sleeping for %v seconds to allow full sync finish", fullSyncWaitTime))
 		time.Sleep(time.Duration(fullSyncWaitTime) * time.Second)
+	} else if serviceName == hostdServiceName {
+		ginkgo.By("Fetch IPs for the all the hosts in the cluster")
+		hostIPs := e2eVSphere.getAllHostsIP(ctx)
+		isServiceStopped = true
+		for index, hostIP := range hostIPs {
+			// Safe desctructive test: Stop hostd service only on half number of ESX Hosts
+			if index%2 == 0 {
```

Comment:
we want to bring down the host where the creation is going on, for the lack of better alternatives we are trying to turnoff hostd on all hosts. If we are skipping few hosts, how can we say the test is achieving its purpose

Reply:
Stopping and starting all the hosts hostd service